src/python/Python-2.7.10/setup.py

```diff
@@ -805,6 +805,9 @@ def detect_modules(self):
                                libraries=math_libs) )
         # Detect SSL support for the socket module (via _ssl)
         search_for_ssl_incs_in = [
+                              # On MacOS OpenSSL headers are installed using
+                              # Homebrew. Make sure they are picked first.
+                              '/usr/local/opt/openssl/include',
                               '/usr/local/ssl/include',
                               '/usr/sfw/include/',
```

Comment:
I know that this was not done in this branch, but maybe we should add a comment for each line from here to describe why we got it here in the first place.. so that we don't have to search the history